subt/doc/virtual-howto.md

```diff
@@ -0,0 +1,61 @@
+# SubT Challenge - Virtual Track HowTo
+
+Here are instructions how to build, run locally and deploy to AWS "robotika" solution docker.
+It is expected that you already have docker installed and your PC has compatible NVidia GPU.
+For official instructions how to start with Docker please look at:
+  https://bitbucket.org/osrf/subt/wiki/tutorials/SystemSetupDockerInstall
+
+
+## Prepare OSGAR build environment
+The convenience scripts for build and run are merged into OSGAR:
+```
+git clone https://github.com/robotika/osgar.git .
+cd ~/osgar/subt/docker
+```
+
+## Download official docker images
+The currently available images are listed at:
+  https://hub.docker.com/r/osrf/subt-virtual-testbed/tags
+Note, that you will need cloudsim_sim_latest, cloudsim_bridge_latest and subt_solution_latest
+```
+docker pull osrf/subt-virtual-testbed:cloudsim_sim_latest
+docker pull osrf/subt-virtual-testbed:cloudsim_bridge_latest
+docker pull osrf/subt-virtual-testbed:subt_solution_latest
+```
+
+## Build and run locally simple unittest image
+```
+cd ~/osgar/subt/docker
+./build.bash unittest
+./run.bash unittest
+```
+
+It is necessary to run also simulation and bridge in other two terminals:
+```
+./run.bash osrf/subt-virtual-testbed:cloudsim_sim_latest cloudsim_sim.ign circuit:=urban worldName:=urban_circuit_practice_01 robotName1:=X0F200L robotConfig1:=X2_SENSOR_CONFIG_3
+./run.bash osrf/subt-virtual-testbed:cloudsim_bridge_latest circuit:=urban worldName:=urban_circuit_practice_01 robotName1:=X0F200L robotConfig1:=X2_SENSOR_CONFIG_3
```

Comment:
Given that the local display error prevents me from actually running it ... Are the three commands going to finish somehow, or am I supposed to kill them?

And what should I expect to happen?